src/site/_includes/components/LanguageList.js

```diff
@@ -16,27 +16,41 @@
 const path = require('path');
 const {getTranslatedUrls, getDefaultUrl} = require('../../_filters/urls');
 const {i18n} = require('../../_filters/i18n');
-const {languageNames} = require('../../../lib/utils/language');
+const {languageNames, scriptGroups} = require('../../../lib/utils/language');
 
 module.exports = (url, lang) => {
-  const langhrefs = getTranslatedUrls(url)
-    .filter((langhref) => langhref[0] !== lang)
-    .map((langhref) => {
-      const href = langhref[1];
-      return `<a class="w-post-signpost__link"
-        translate="no"
-        lang="${langhref[0]}"
-        href="${href}">
-      ${languageNames[langhref[0]]}</a>`;
-    });
+  let langhrefs = getTranslatedUrls(url).filter(
```

Comment:
I also didn't want to have to change this from `const` to `let`, but since the previous logic was adding the "en" data to `langhrefs` after the initial .filter() and .map() of the getTransledUrls(url) array, it wasn't apart of the sorting when I added the sort logic here initially. So my thought process was to keep the original .filter() call and then do a quick length check before adding the "en" data to `langhrefs` and after this begin the sorting before finally using .map() to format the strings into HTML tags.